other/check-vpa-configuration/check-vpa-configuration.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: check-vpa-configuration
+  annotations:
+    policies.kyverno.io/title: Check for matching VerticalPodAutoscaler (VPA)
+    policies.kyverno.io/category: Availability
```

```suggestion
    policies.kyverno.io/category: Other
```

Does not need a new category.